Plugins/HockeyAppUnityAndroid/HockeyAppUnity-Scripts/HockeyAppAndroid.cs

```diff
@@ -307,9 +291,6 @@ protected virtual List<string> GetLogHeaders ()
 		string model = GetModel();
 		list.Add("Model: " + model);
 
-		string crashReporterKey = GetCrashReporterKey();
-		list.Add("CrashReporter Key: " + crashReporterKey);
```

Comment:
I think `DEVICE_IDENTIFIER` should be used here instead of removing. Like [here](https://github.com/bitstadium/HockeySDK-Android/blob/develop/hockeysdk/src/main/java/net/hockeyapp/android/ExceptionHandler.java#L189) in native SDK.

Reply:
@MatkovIvan thx for catching this!

@bmourat can you create a new PR, please?